packages/usage-examples/src/mock/mock-storage.ts

```diff
@@ -10,7 +7,9 @@ import { EventEmitter } from 'events';
  * Storage layer implemented with in-memory hashmap, to be used for testing.
  */
 export default class MockStorage implements StorageTypes.IStorage {
```

Comment:
Just saw in this PR we have mock-storage defined thrice (Here, in client, and in prototype-estimator).
Maybe we could do some factoring